packages/flame/test/experimental/layout_component_test.dart

```diff
@@ -0,0 +1,287 @@
+import 'package:flame/components.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/game.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  late Component component;
+  late Vector2 gameSize;
+
+  late TextComponent firstText;
+  late TextComponent secondText;
+  late TextComponent thirdText;
+  late double componentsWidth;
+  late double componentsHeight;
+
+  void _initChildren() {
+    firstText = TextComponent(
+      text: 'First',
+    )
+      ..textRenderer = TextPaint(
+        style: const TextStyle(
+          fontSize: 22,
+        ),
+      )
+      ..size = Vector2(50, 50);
```

Comment:
You don't need a `TextComponent` here: just a simple `PositionComponent(size: Vector2(50,50))` will do.
(Similarly for secondText and thirdText).